tools/custom-action/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash --login
+set -eo pipefail
+
+. /prepare.sh # Execute the prepare script
+git init # To ensure subsequent git commands pick the workspace
+git config --global --add safe.directory /github/workspace
+BIKESHED_DISALLOW_ONLINE=1 REQUIRE_DIAGRAM_GENERATION=1 make -j out
+bash tools/check-repo-clean.sh
```

Comment:
I think we should leave this disabled for now because we still have the problem that it's impossible to generate the same version locally unless you use the devcontainer. Which I don't expect everyone to do.

Reply:
When I try local and devcontainer, it is the same. Odd one was GitHub Actions direct execution

Reply:
Developers' local machines will also be different though if they don't use the devcontainer.

Reply:
But so far devcontainer was same as my local generation (the only time I saw different generation was with GitHub Actions). Do you get errors with the check on your local setup?

Reply:
I don't know but if GitHub Actions is different I don't know why all user machines would be the same. Maybe we can just leave this and investigate if someone runs into a problem.

Reply:
Sounds like a good plan to me, having the check on until we hear an issue is probably a bit safer. Fortunately we don't run the different generation case of Actions' environment anymore